### ISSUE TITLE
Question - registerNotificationReceivedBackground on Android

### ISSUE THREAD

**Issue by mjmostachetti** [wix/react-native-notifications#952]
I'm on 4.3.3 of this package and 67.5 for React Native.

When I register for registerNotificationReceivedBackground it works as expected - I put the app in the background and receive notifications, even if I close the app. When I close the app and open the app again and reregister, it fires twice now. If I unregister when the app closes, then I don't receive notifications in the background anymore. Am I missing something or is this expected behaviour? Thanks.

**Comment by mjmostachetti** [comment]
Just to test, I created a new project with just this package and the bare-bones latest RN code.
RN - 0.71.3
RNN - 4.3.3

It was kind of interesting. I did not dispose any of the EventEmitter returned from the .events(). I no longer saw the +1 issue when restarting the application, although once the application closed, I no longer received notifications with the application closed.

I made sure to follow the instructions in the Android setup, and it seems like including this line is out-of-date
classpath 'com.google.gms:google-services:4.0.0'
